server/nuxeo-launcher/src/main/java/org/nuxeo/launcher/NuxeoLauncher.java

```diff
@@ -2267,15 +2272,20 @@ protected boolean pkgReset() {
      */
     protected void printInstanceXMLOutput(InstanceInfo instance) {
         try {
-            JAXBContext jaxbContext = JAXBContext.newInstance(InstanceInfo.class, DistributionInfo.class,
-                    PackageInfo.class, ConfigurationInfo.class, KeyValueInfo.class);
-            printXMLOutput(jaxbContext, instance, System.out);
+            printInstanceXMLOutput(instance, System.out);
         } catch (JAXBException | XMLStreamException | FactoryConfigurationError e) {
             throw new NuxeoLauncherException("Output serialization failed: " + e.getMessage(), EXIT_CODE_NOT_RUNNING,
                     e);
         }
     }
 
+    protected void printInstanceXMLOutput(InstanceInfo instance, OutputStream out)
```

Comment:
if it's been extracted for being able to choose the outputStream easily in the test, maybe I would indicate it in a comment. Or is it going to be used and reused?

Reply:
In fact, this was present in 10.10 but removed in 11 and now back for test yes. I'm not sure it needs a comment.